sqlglot/expressions.py

```diff
@@ -344,7 +344,7 @@ def to_s(self, hide_missing=True, level=0):
 
         return indent + left + right
 
-    def transform(self, fun, *args, copy=True, **kwargs):
+    def transform(self, fun, *args, copy=True, allow_removals=False, **kwargs):
         """
```

Comment:
i think we can get rid of the allow removals arg

@barakalon what so you think

Reply:
Shouldn't we keep this info local to `transform`? Since it's not something we would use inside our callback.

Reply:
what do you mean? i'm saying we could just always allow_removals?

Reply:
Oh ok, I thought you meant to just do `if kwargs.get("allow_removals") ...` instead.

Reply:
Yeah I think we should always allow removals.